checkov/github_actions/runner.py

```diff
@@ -144,3 +144,14 @@ def extract_images(
             images.extend(manager.extract_images_from_workflow())
 
         return images
+
+    def populate_metadata_dict(self) -> None:
+        if self.check_type == CheckType.GITHUB_ACTIONS and isinstance(self.definitions, dict):
```

Comment:
This is the GHA runner...
```suggestion
        if isinstance(self.definitions, dict):
```